src/dsp_tools/xmllib/models/dsp_base_resources.py

```diff
@@ -58,7 +58,7 @@ def add_comment(self, comment: str) -> AnnotationResource:
         self.comments.append(comment)
         return self
 
-    def add_comments(self, comments: list[str]) -> AnnotationResource:
+    def add_comment_multiple(self, comments: list[str]) -> AnnotationResource:
```

Comment:
I would strongly prefer `add_multiple_comments` then. Because the current name is not naturally understandable

Reply:
I asked her about that, and she does not like it because when you start typing you first think about add_comment and then you think about if it is optional or multiple.

Reply:
That's a good point, indeed. In this case, let's listen to our users :-)